wordpress-mobile/WordPress-Android
# Change page list labels (private, etc) to be gray

Similiar to [this](https://github.com/wordpress-mobile/WordPress-iOS/pull/12259) iOS PR, we are changing page labels from yellow to gray to improve accessibility.


Before | After
----- | -----
| <img width="366" alt="a" src="https://user-images.githubusercontent.com/728822/64054625-c7ad2400-cb3c-11e9-808a-ff4365f0f29f.png"> | <img width="365" alt="b" src="https://user-images.githubusercontent.com/728822/64054626-c7ad2400-cb3c-11e9-9ab2-6fbd2bd97cbf.png"> |

To test:

1. Open the pages list, find or create a private page
2. Ensure that the new appearance matches the description

Update release notes:
